modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -97,6 +97,29 @@ definition {
 			value1 = "Save");
 	}
 
+	macro createAPIEndpointInEditAPIApplicationPage {
```

Comment:
Please try to use the existing createAPIEndpoint() instead of adding a new one, thanks!